manim/cli/default_group.py

```diff
@@ -0,0 +1,71 @@
+"""DefaultGroup allows a subcommand to act as the main command
+
+In particular, this class is what allows `manim` to act as `manim render`.
+"""
+import cloup
+
+from .. import logger
+
+__all__ = ["DefaultGroup"]
+
+
+class DefaultGroup(cloup.Group):
+    """Invokes a subcommand marked with `default=True` if any subcommand not
+    chosen.
+
+    :param default_if_no_args: resolves to the default command if no arguments
```

Comment:
This won't produce the correct formatting, you have to do something like
```
Parameters
----------
default_if_no_args
    parameter description ...
```

However, this currently seems a bit odd to me: `default_if_no_args` is not an explicit parameter of the classes `__init__`.

Reply:
I will remove the mention of `default_if_no_args` as it's mentioned in the parent class.